### PR TITLE
Add progress-indicator emojis to explainer list

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Description

- Added emoji 'progress' indicators to explainers, so it's easy to see at a glance the status of different features. I don't know if my emoji choices actually make sense, or if we want to have text as well. I went with 👍🏼 for CSSWG approval, proposals moving forward; 🚀 for features that are starting to ship in browsers; ✅ for fully implemented features; and ❌ for abandoned features.
- Added a note to the mixins/functions explainer, linking to the CSSWG adoption resolution.

## Show me

Screenshot 2024-03-17 at 11 00 39 AM

### CODE REVIEW

**Review by dvdherron — APPROVED** [review]
I like the idea of indicating progress with emojis. Maybe on the overview page it would be helpful with a short and sweet legend?

File: src/_data/status.yaml [hunk 1]
```diff
@@ -4,15 +4,19 @@ section:
...
+    ❌
     This feature has been abandoned
     (at least for now).
     Our notes are likely out-of-date.
   shipped: >
+    ✅
     This feature has shipped in browsers.
     Our notes are likely out-of-date.
   moved: >
+    👀
```
Comment (stacyk) [inline src/_data/status.yaml:16]: Should we use the 🚚 emoji for moved? https://gitmoji.dev

File: src/overflow/explainer.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 created: 2023-12-14
 title: CSS Overflow - Broad Research
+progress: 👍🏼
```
Comment (stacyk) [inline src/overflow/explainer.md:4]: I think the "👍" is preferred for screen readers as it just reads "bullet thumbs up" as opposed to "bullet thumbs up medium light skin tone"

**Comment by mirisuzanne** [comment]
@stacyk @dvdherron see what you think about that update. The other choice I'm questioning here is 🚀 for starting to ship. Open to ideas.

**Review by dvdherron — APPROVED** [review]
changes LGTM. (And I think the 🚀 works for shipping)

**Review by stacyk — APPROVED** [review]
Looks good! I also think rocket is good for shipped...but 🚢 could also work? Screenreader will annouce it as "ship" so it's close ¯\_(ツ)_/¯

**Pull request merged**